### PR TITLE
update  manage-resources documentation

### PR DESCRIPTION
**Pull request by TinySong**
1. Update doc URL
2. Update misleading sentence.
3. Formatting sentences with only one line.

### CODE REVIEW

**Comment by TinySong** [comment]
@jimangel @zacharysarah Sorry to disturb you, but Could you have time to take look at this PR ?

File: content/en/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace.md [hunk 1]
```diff
@@ -214,7 +214,10 @@ applied the
 [default CPU request and limit](/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace/)
 from the LimitRange for this namespace.
 
-At this point, your Pod might be running or it might not be running. Recall that a prerequisite for this task is that your cluster must have at least 1 CPU available for use. If each of your Nodes has only 1 CPU, then there might not be enough allocatable CPU on any Node to accommodate a request of 800 millicpu. If you happen to be using Nodes with 2 CPU, then you probably have enough CPU to accommodate the 800 millicpu request.
+At this point, your Pod might be running or it might not be running. Recall that a prerequisite for
```
Comment (zacharysarah) [inline content/en/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace.md:217]:
```suggestion
At this point, your Pod may or may not be running. Recall that a prerequisite for
```

File: content/en/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace.md [hunk 2]
```diff
@@ -76,8 +77,8 @@ file for the LimitRange, they were created automatically.
 Now whenever you define a Pod within the constraints-mem-example namespace, Kubernetes
 performs these steps:
 
-* If any container in that Pod does not specify its own memory request and limit, assign
-the default memory request and limit to that container.
+* If any container in that Pod does not specify its own memory request and limit, 
+the control plane assign the default memory request and limit to that container.
```
Comment (zacharysarah) [inline content/en/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace.md:81]:
```suggestion
the control plane assigns the default memory request and limit to that container.
```

**Review by zacharysarah — APPROVED** [review]
@TinySong Thanks, these are really helpful changes. I like the improved clarity!

Some grammar nits, otherwise LGTM.

**Comment by zacharysarah** [comment]
Approving this PR to move it forward, adding an explicit hold for @TinySong to address review feedback.

Other reviewers, feel free to `/lgtm`.

/approve
/hold

**Pull request merged**